source/Application/Controller/Admin/ShopLicense.php

```diff
@@ -101,8 +102,11 @@ protected function _fetchCurVersionInfo($sUrl)
         $oCurl->setMethod("POST");
         $oCurl->setUrl($sUrl . "/" . $sLang);
         $oCurl->setParameters($aParams);
-        $sOutput = $oCurl->execute();
-
+        try {
+            $sOutput = $oCurl->execute();
+        } catch (ConnectionException $ex) {
+            $sOutput = "Connecting to $sUrl failed, please check network and firewalls";
```

Comment:
```suggestion
            $sOutput = "<div class='errorbox'>Connecting to $sUrl failed, please check network and firewalls</div>";
```

Would be even better to extraxt the message to a language-string like this:
```suggestion
            $sOutput = '<div class="errorbox">';
            $sOutput .= sprintf($oLang->translateString('LICENCE_CHECK_FAILED', $iLang, true), $sUrl);
            $sOutput .= '</div>';
```

Reply:
![image](https://user-images.githubusercontent.com/1001186/49872341-0c5f2680-fe19-11e8-9eb4-f16abdfa0251.png)